README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
To be honest I forgot what timeout does when given nil/0 because I don't use it. The initial issue still brings up a good point, that the documentation needs some improvement. I've been meaning to put some work into that before the next major release of circuitbox.